src/sonic-yang-models/yang-templates/sonic-acl.yang.j2

```diff
@@ -152,7 +152,7 @@ module sonic-acl {
 
 				leaf ETHER_TYPE {
 					type string {
-					    pattern "0x0[6-9a-fA-F][0-9a-fA-F]{2}|0x[1-9a-fA-F][0-9a-fA-F]{3}";
+                                            pattern "0x0[6-9a-fA-F][0-9a-fA-F]{2}|0x[1-9a-fA-F][0-9a-fA-F]{3}|153[6-9]|15[4-9][0-9]|1[6-9][0-9][0-9]|[2-9][0-9]{3}|[1-5][0-9]{4}|6553[0-5]|655[0-2][0-9]|65[0-4][0-9]{2}|6[0-4][0-9]{3}";
```

Comment:
since both dec and hex are all supported, is it possible to validate against the range [0, 65535]?

Reply:
Hi Kebo,
I believe the intention is to restrict the ethertypes to ethernet specific values which start from 1536 or 0x600
https://www.iana.org/assignments/ieee-802-numbers/ieee-802-numbers.xhtml. So I didn't want to change the behavior.